### DISCUSSION TITLE
Can I control response headers?

### DISCUSSION THREAD

**Discussion by songjiyeong** [grafana/grafana-infinity-datasource#588] [General]
Hi. 
I'm trying to prove it using the API Key and get the information by receiving the x-auth-token in the response headers. I only see the body part of the response. Can I control the response headers?

**Comment by yesoreyeram** [comment]
What do you mean by control? How can one control the response headers? Plugin is responsible only for request headers

**Comment by songjiyeong** [comment]
Did I ask a stupid question? :)
There are many unknowns when trying to express it in Grafana.
I asked a question while thinking about how to reuse the x-auth-token that
comes with the response header to send a request.
Anyway your plugin is awesome. That's a lot of help.

2023년 4월 28일 (금) 오후 3:53, Sriram ***@***.***>님이 작성:
